honeybee_energy/ventcool/afn.py

```diff
@@ -0,0 +1,391 @@
+        opening_area += sum([door.area for door in ext_wall.doors])
+        face_area = ext_wall.area - opening_area
+        flow_cof = ext_cracks['wall_flow_cof'] * face_area
+        flow_exp = ext_cracks['wall_flow_exp']
+        ext_wall.properties.energy.vent_crack = AFNCrack(flow_cof, flow_exp)
+
+    for ext_roof in ext_roofs:
+        opening_area = sum([aper.area for aper in ext_roof.apertures])
+        face_area = ext_roof.area - opening_area
+        flow_cof = ext_cracks['roof_flow_cof'] * face_area
+        flow_exp = ext_cracks['roof_flow_exp']
+        ext_roof.properties.energy.vent_crack = AFNCrack(flow_cof, flow_exp)
+
+    # add exterior opening leakage components
+    for ext_aperture in ext_apertures:
+        if ext_aperture.properties.energy.vent_opening is None:
+            ext_aperture.is_operable = True
+            ext_aperture.properties.energy.vent_opening = VentilationOpening()
+        vent_opening = ext_aperture.properties.energy.vent_opening
+        vent_opening.flow_coefficient_closed = ext_cracks['window_flow_cof']
+        vent_opening.flow_exponent_closed = ext_cracks['window_flow_exp']
+
+    for ext_door in ext_doors:
+        if ext_door.properties.energy.vent_opening is None:
+            ext_door.properties.energy.vent_opening = VentilationOpening()
+        vent_opening = ext_door.properties.energy.vent_opening
+        vent_opening.flow_coefficient_closed = ext_cracks['door_flow_cof']
+        vent_opening.flow_exponent_closed = ext_cracks['door_flow_exp']
+
+
+def generate(rooms, window_vent_controls, leakage_type='Average',
+             use_room_infiltration=True, atmospheric_pressure=101325):
+    """
+    Mutate a list of Honeybee Room objects to represent an EnergyPlus AirflowNetwork.
+
+    This function will compute leakage component parameters for the ventilation
+    cooling energy properties of Honeybee Room and Face objects to simulate an
+    EnergyPlus AirflowNetwork.
+
+    VentilationOpening objects will be added to Aperture and Door objects if not already
+    defined. If already defined, only the parameters defining leakage when the openings
+    are closed will be overwritten. AFNCrack objects will be added to all external and
+    internal Face objects, and any existing AFNCrack objects will be overwritten.
+
+    Args:
+        rooms: List of Honeybee Room objects that make up the Airflow Network. The
+            adjacencies of these rooms must be solved.
+        window_vent_controls: List or tuple of VentilationControl objects or None,
+            corresponding to the list of rooms.
+        leakage_type: Text identifying the leakiness of the internal walls. This
+            will be used to determine the air mass flow rate parameters for cracks in
+            internal floors, ceilings and walls. (Default: 'Average').
+            Choose from the following:
+
+            * Tight
+            * Average
+            * Leaky
```

Comment:
I assume this comes from the Design Builder templates?  If so, can you add a `Note:` section with a reference like we do in the ladybug-comfort docstrings:
https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/pmv.py#L19-L30